### PR TITLE
Added regex replace for media download

### LINKED ISSUE #1337: File Download not working with special characters
When uploading an image with special characters like the german Umlaute, the download isn't working. Example filename: `wöchentlicher_plan.pdf`

bildschirmfoto 2015-07-03 um 10 10 38

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1337
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php [hunk 1]
```diff
@@ -120,10 +119,12 @@ protected function getFileResponse($fileVersion, $dispositionType = ResponseHead
             fclose($handle);
         });
 
+        $pathInfo = pathinfo($fileName);
+
         // Prepare headers
         $disposition = $response->headers->makeDisposition(
             $dispositionType,
-            basename($fileName)
+            preg_replace('/[^A-Za-z0-9\-]/', '', $pathInfo['filename']) . '.' . $pathInfo['extension']
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php]: Just an idea, why don't we use the `PathCleanup` service here? Would probably provide better URLs.

**Comment by danrot** [comment]
Tests missing.

File: src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php (hunk 1 above)
Comment (alexander-schranz) [inline src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php]: @wachterjohannes  don't overwrite the second parameter
set the third parameter with the regex!

Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php]: @danrot i have also thought about it ... perhaps its a good idea (:

**Pull request merged**